weasel/__main__.py

```diff
@@ -0,0 +1,3 @@
+from ._util import COMMAND, app
+
+app(prog_name=COMMAND)
```

Comment:
Is it convention to have this in `__main__`?

Reply:
I'm not totally sure what you mean... Putting `app()` there is how `python -m <package>` operates: https://typer.tiangolo.com/tutorial/package/#add-a-__main__py